Makefile

```
@@ -52,6 +47,9 @@ php: ## Open a terminal in the "php" container
 ps: ## List all containers managed by the environment
 	docker-compose ps
 
+restart: ## Restart the environment
```

Would it work like this?

```suggestion
restart: stop start ## Restart the environment
```